internal/local/dependencies.go

```diff
@@ -38,3 +38,31 @@ func FindAppDependencies(path string) (Dependencies, error) {
 
 	return Dependencies{rootDir, archivePath}, nil
 }
+
+// FindPackageJSON finds the Realm app dependencies JSON
+func FindPackageJSON(path string) (Dependencies, error) {
+	app, appOK, appErr := FindApp(path)
+	if appErr != nil {
+		return Dependencies{}, appErr
+	}
+	if !appOK {
+		return Dependencies{}, nil
+	}
+
+	rootDir := filepath.Join(app.RootDir, NameFunctions)
+
+	JSONs, err := filepath.Glob(filepath.Join(rootDir, nameJSON))
```

Comment:
[nit] can we make this `JSONPaths`, I think it's a little clearer

Reply:
I don't think you need to call `filepath.Glob` anymore. You can just do:
```
jsonPath, err := filepath.Abs(filepath.Join(rootDir, nameJSON))
```